skbio/sequence/_sequence.py

```diff
@@ -886,17 +880,17 @@ def __getitem__(self, indexable):
         seq = self._bytes[indexable]
         positional_metadata = self._slice_positional_metadata(indexable)
 
-        return self._to(sequence=seq, positional_metadata=positional_metadata)
+        return self._constructor(
+            sequence=seq,
+            metadata=self.metadata,
```

Comment:
Does `Sequence._constructor` not propagate attributes automatically? This was one of the reasons for `_to`. I think `TabularMSA` propagates.